doc/mob_db_mode_list.txt

```diff
@@ -80,6 +83,14 @@ NoKnockback: Monsters will be immune to Knockback's.
 Random Target: Picks a new random target in range on each attack / skill.
 	(not implemented)
 
+No Random Walk: Monsters will not randomly walk around during idle state.
```

Comment:
if this is non official modes, probably better add comment about this in docs

Reply:
no random walk and no cast skill is official, I can quite confirm this is in wave dungeon
https://irowiki.org/wiki/Wave_Mode
however can attack mob is unofficial ...
so just add (unofficial) ?